doc/source/tutorials/pidloops.rst

```diff
@@ -331,13 +366,13 @@ The script we'll use to tune the highly overpowered rocket shown will launch the
     WAIT 3.
 
     UNTIL SHIP:ALTITUDE > 30000 {
-        SET dt TO TIME:SECONDS - t0.
-        IF dt > 0 {
-            SET thrott TO MIN(1,MAX(0,thrott + dthrott)).
-            SET t0 TO TIME:SECONDS.
-            LOG logline TO throttle_log.
-        }
-        WAIT 0.001.
+	SET dt TO TIME:SECONDS - t0.
+	IF dt > 0 {
+	    SET thrott TO MIN(1,MAX(0,thrott + dthrott)).
+	    SET t0 TO TIME:SECONDS.
+	    LOG logline TO throttle_log.
+	}
+	WAIT 0.001.
```

Comment:
Irrelevant whitespace diff because the original was written using tabs and I have my editor set to change tabs to spaces.